ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -64,12 +64,9 @@ public extension Colors {
     internal struct Table {
```

Comment:
Isn't this entire internal struct "Table" supposed to be replaced by tokens?

Looks like only the following are still used in code:

- Colors.Table.Cell.accessoryDisclosureIndicator
- Colors.Table.Cell.accessoryDetailButton